src/js/main.js

```diff
@@ -631,4 +633,130 @@ Site.Player = {
   },
 };
 
+Site.EventChecker = {
+  eventUrl: WP.siteUrl + '/wp-json/igv/current-event',
+  eventCheckInterval: 5000,
+  eventData: {},
+  init: function() {
+    var _this = this;
+
+    // Event title element
+    _this.eventTitle = document.getElementById('event-title');
+
+    // Set viewport orientation
+    _this.setViewportLargestDimension();
+
+    // Bind orientationchange
+    _this.setViewportLargestDimension = _this.setViewportLargestDimension.bind(_this);
+
+    // Start checker
+    _this.startChecker();
+  },
+
+  setViewportLargestDimension: function() {
+    var _this = this;
+
+    // Check if height is larger than width
+    if (window.innerHeight > document.body.clientWidth) {
+      _this.viewporLargestDimension = {
+        dimension: 'height',
+        size: window.innerHeight
+      };
+
+    } else {
+      _this.viewporLargestDimension = {
+        dimension: 'width',
+        size: document.body.clientWidth,
+      };
+
+    }
+
+  },
+
+  startChecker: function() {
+    var _this = this;
+
+    // Check the current event
+    _this.checkEvent();
+
+    // Start checker on an interval
+    setInterval(function() {
+      _this.checkEvent();
+    }, _this.eventCheckInterval);
+  },
+
+  // Check the current/next event and update background image and show title
+  checkEvent: function() {
+    var _this = this;
+
+    // Make the ajax request
+    $.getJSON(_this.eventUrl, function(data) {
+      if (data) {
+
+        // Check if new data is different from current one
+        if (_this.eventData['title'] !== data['title']) {
+
+          // Save event data
+          _this.eventData = data;
+
+          // Set background image
+          _this.setBackground(data.featured_thumbnails);
+
+          // Set current/next show title
+          _this.eventTitle.innerHTML = data.title;
+        }
+      } else {
+
+        // Set event data to false
+        _this.eventData = false;
+
+        // Clear background
+        _this.clearBackground();
+      }
+    });
+  },
+
+  setBackground: function(images) {
+    var _this = this;
+
+    var viewportLargestDimension = _this.viewporLargestDimension.size
+
+    // We set largest image as fallback (last in the array)
+    var background = images[images.length - 1].url;
```

Comment:
needs a condition for when featured image not set. should fallback to Open Graph default and then to nothing.